README.md

```diff
@@ -119,6 +119,7 @@ Table of Contents
   * http://vaddy.net/ - Continuous web security testing with continuous integration (CI) tools. 3 domains, 10 scan history for free
   * https://www.globalsign.com/en/ssl/ssl-open-source/ - Free SSL certs for Open Source projects
   * https://www.startssl.com/ - Free SSL certs
+  * https://buy.wosign.com/free/ - Free SSL certs. Up to 100 subdomains. China authority
```

Comment:
It disable free 100 domains/subdomains for certificate. 
Now free certificate avalaible for one domain only (+www subdomain - free too), more domains have a small cost.